kuma/search/signals.py

```diff
@@ -22,7 +22,7 @@ def render_done_handler(**kwargs):
         if outdated:
             log.info('Found a newer index and scheduled '
                      'indexing it after promotion.')
-        doc_pks = set(doc.other_translations.values_list('pk', flat=True))
+        doc_pks = set([item.pk for item in doc.other_translations])
```

Comment:
The previous version looked like it would work and be more efficient.  ``values_list`` just returns the PKs, where the second version loads then entire ``Document`` record from the database, for the same effect.  Did the previous version have the wrong result?